src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,51 @@
+package coffee.order;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoffeeOrderBoard {
+    private List<Order> orders;
+
+    public CoffeeOrderBoard() {
+        this.orders = new ArrayList<>();
+    }
+
+
+    public void add(Order order) {
+        if (order == null) {
+            throw new IllegalArgumentException("Order cannot be null");
+        }
+        order.setNumber(orders.size() + 1);

```

Comment:
Would be better to generate `orderNumber` using a separate variable, not the size of the `orders`.